tests/fixtures/tests.py

```diff
@@ -619,6 +619,39 @@ def test_output_formats(self):
             format='xml', natural_foreign_keys=True
         )
 
+    def test_loading_with_excludes(self):
+        Site.objects.all().delete()
+
+        management.call_command('loaddata', 'fixture1', exclude=['fixtures'], verbosity=0)
+        self.assertFalse(Article.objects.exists())
+        self.assertFalse(Category.objects.exists())
+        self.assertQuerysetEqual(Site.objects.all(), [
+            '<Site: example.com>',
+        ])
+
+        Site.objects.all().delete()
+        Article.objects.all().delete()
+        Category.objects.all().delete()
+
+        management.call_command('loaddata', 'fixture1', exclude=['fixtures.Article'], verbosity=0)
+        self.assertFalse(Article.objects.exists())
+        self.assertQuerysetEqual(Category.objects.all(), [
+            '<Category: News Stories>',
+        ])
+        self.assertQuerysetEqual(Site.objects.all(), [
+            '<Site: example.com>',
+        ])
+
+        # Excluding a bogus app should throw an error
+        with six.assertRaisesRegex(self, management.CommandError,
```

Comment:
Should be replaced with [`assertRaisesMessage`](https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.SimpleTestCase.assertRaisesMessage).